lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -1153,6 +1176,8 @@ def check_host():
             equal(self.param('io_threads'), entity.io.threads) and
             equal(self.param('ballooning_enabled'), entity.memory_policy.ballooning) and
             equal(self.param('serial_console'), entity.console.enabled) and
+            equal(self._get_minor(self.param('custom_compatibility_version')), self._get_minor(entity.custom_compatibility_version)) and
```

Comment:
I think you forgot to push the change to `ovirt.py` where you add methods `_get_minor` and `_get_major`